Remove the dependency on utils/Callstack from libhidl/base/Status.cpp.

We will not be implementing the android backtrace library, so using Callstack
represents no value. Remove it which also breaks the dependency on back trace.

diff --git a/platform2/aosp/system/libhidl/base/Status.cpp b/platform2/aosp/system/libhidl/base/Status.cpp
index 41c9640..c04c14b 100644
--- a/platform2/aosp/system/libhidl/base/Status.cpp
+++ b/platform2/aosp/system/libhidl/base/Status.cpp
@@ -18,7 +18,6 @@
 #include <android-base/logging.h>
 
 #include <hidl/Status.h>
-#include <utils/CallStack.h>
 
 #include <unordered_map>
 
@@ -131,8 +130,6 @@ namespace details {
 
         if (gReturnRestriction == HidlReturnRestriction::ERROR_IF_UNCHECKED) {
             LOG(ERROR) << "Failed to check status of HIDL Return.";
-            CallStack::logStack("unchecked HIDL return", CallStack::getCurrent(10).get(),
-                                ANDROID_LOG_ERROR);
         } else {
             LOG(FATAL) << "Failed to check status of HIDL Return.";
         }
